sdks/java/core/src/main/java/org/apache/beam/sdk/values/PValue.java

```diff
@@ -18,13 +18,18 @@
 package org.apache.beam.sdk.values;
 
 import java.util.Map;
+import org.apache.beam.sdk.annotations.Internal;
 import org.apache.beam.sdk.transforms.PTransform;
 
 /**
- * The interface for values that can be input to and output from {@link PTransform PTransforms}.
+ * <b><i>For internal use. No backwards compatibility guarantees.</i></b>
+ *
+ * <p>The interface for primitive values that can be input to and output from
```

Comment:
Since this paragraph is tautological and the next is about backwards compat (for which there is none), consider deleting them. Anyone who doesn't "already know" what `PValue` is for should not be touching it.